### PR TITLE
Add LG OnScreenControl

### PR DESCRIPTION
**Pull request by bwilczynski**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/onscreen-control.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'onscreen-control' do
+  version '1.37'
+  sha256 '7263b8cb714cf25db16cee7310ba03a79cc28754cb18f55c30642cc745d64740'
+
+  url "http://www.lg.com/us/lgecs.downloadFile.ldwf?DOC_ID=20150116943155&what=MANUAL&fromSystem=LG.COM&fileId=kF2MFoPcI9YSuCcD23D9w&ORIGINAL_NAME_b1_a1=MAC_OnScreenControl.#{version}.zip"
+  name 'LG OnScreen Control'
+  homepage 'http://www.lg.com/'
```
Comment (vitorgalvao) [inline Casks/onscreen-control.rb]: This is really generic. It needs to be more specific.

What’s this app? What does it do?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/onscreen-control.rb (hunk 1 above)
Comment (bwilczynski) [inline Casks/onscreen-control.rb]: This app allows you to do a screen split on LG displays, but also changes display settings (like brightness, constrast etc.) If you think this needs to go to drivers I will move it there.

Unfortunatelly there is no homepage for this app, even clicking on a "home" button inside the app redirects you to LG homepage.

Comment (vitorgalvao) [inline Casks/onscreen-control.rb]: Definitely belongs in drivers, yes, since it’s useless without and LG display. Please resubmit there.

Comment (bwilczynski) [inline Casks/onscreen-control.rb]: https://github.com/caskroom/homebrew-drivers/pull/6

**Pull request closed without merging**